_posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.3.0"
+authors:
+- dlv
+- kkondaka
+date: 2023-05-25 21:30:00 -0700
+categories:
+  - releases
+meta_keywords: Data Prepper 2.3.0
```

Comment:
@pajuric , Any suggestions here?

Some things that come to mind are "S3", "tail sampling"

Reply:
`ingestion, observability, log pipelines` might be some good ones.

Reply:
I can take a look at the meta keywords and description. I'll drop them into the request shortly.

Reply:
Looking at keywords. I want to be sure that what we're using is anchored by the text - and we should try to limit it to no more than 3 keywords or phrases. Here's my suggestions:


**meta_keywords:** Data Prepper expressions, tail sampling in Data Prepper, add entries processor
**meta_description:** Learn how Data Prepper 2.3.0 improves the ability to process complex expressions, supports event tagging, and enhances the add entries processor. Download it now.